vendor/magento/zendframework1/library/Zend/Locale/Data/root.xml

```diff
@@ -0,0 +1,3771 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
We should not push changes on the vendor folder, that is handled by composer.

Reply:
Then how can I update the changes related to the vendor?